src/components/include/application_manager/application_manager.h

```diff
@@ -296,6 +296,18 @@ class ApplicationManager {
                                  mobile_apis::HMILevel::eType from,
                                  mobile_apis::HMILevel::eType to) = 0;
 
+  /**
+   * @brief Updates streaming service status for specified session and notifies
+   * HMI via notification if required
+   * @param service_type Id of service which status should be updated
+   * @param app_id Id of session which status should be updated
+   * @param status New streming status for specified session
+   */
+  virtual void ProcessOnDataStreamingNotification(
+      const protocol_handler::ServiceType service_type,
+      const uint32_t app_id,
+      const bool status) = 0;
```

Comment:
@mkorniichuk better names for this param is `should_be_inserted`, `insert_required`, `is_new_service` etc.. Otherwise you can declare enum class describing actions should be done by your function (`kAddService`, `kRemoveService`). 
BTW this will make your code much cleaner as `kAddService` is more descriptive than just true or false while passing the params.

Reply:
@AKalinich-Luxoft I've renamed this parameter in 112037801c58a44cf62a8d078ab186fb9f25e7b5.
As for me, it is better to use a variable, cause it stands for the availability of streaming data. And enum, from my perspective, is redundant.